flink-python/docs/pyflink.rst

```diff
@@ -26,6 +26,9 @@ Subpackages
     :maxdepth: 1
 
     pyflink.table
+    pyflink.batch
```

Comment:
the  index.rst  should also add the module name:
- pyflink.common
- pyflink.dataset
- pyflink.datastream